src/filtering/filterDialogue.component.test.tsx

```diff
@@ -118,4 +118,94 @@ describe('Filter dialogue component', () => {
 
     expect(screen.getByText('Apply')).toBeDisabled();
   });
+
+  it('adds new filter when Add new filter button is clicked and renders multiple filters', async () => {
+    const user = userEvent.setup();
+
+    createView();
+
+    await user.click(screen.getByText('Add new filter'));
+
+    expect(screen.getAllByRole('combobox', { name: 'Filter' })).toHaveLength(2);
```

Comment:
Could we do a check for what happens when the Apply button is clicked here as well? i.e. check the store holds both filters

Reply:
This test is just about that it adds an additional filter box though. If I just press `Apply` here, the store won't update as they're both blank and we remove "extra" blanks. I could modify the `delete a filter` test so that there's three initial filters and that after I delete one there's two instead? Otherwise this test would have to be much longer as I would have to type out the filters to get them to save...

Reply:
I test that you can apply multiple filters in the e2e tests as well

Reply:
Right, my meaning was to actually extend the test and add multiple, actual filters to the store. I do think it's a worthwhile test to have

Reply:
Okay, I'll do  that then